### ISSUE TITLE
feature request: rounded corners

### ISSUE THREAD

**Issue by ZoeFiri** [herbstluftwm/herbstluftwm#552]
This is a feature which I've seen growing in demand in recent times, and can now be had in a great variety of tiling WMs like awesome, i3, and bsp. I find it to be very attractive myself and was wondering if something like this is on the roadmap atm?

**Comment by t-wissmann** [comment]
No it's not on the current roadmap currently.

**Comment by acxz** [comment]
I'm also interested in this feature. Here is a fork that implements rounded corners: https://github.com/calvinhenderson/herbstluftwm/tree/round-corners

Here is the specific commit, although maybe the hlwm codebase has changed: https://github.com/calvinhenderson/herbstluftwm/commit/b8a78e7e77126bbc25369459827b1ba51154b466

Here is an example screenshot: https://i.redd.it/eg997tik5re31.png

**Comment by t-wissmann** [comment]
Exciting! How does your fork interact with the other border settings? E.g. what happens if the border color and the background color of the window is not the same?

**Comment by acxz** [comment]
Thx for responding so fast. It’s actually not my fork. To be frank I haven’t played around with it yet, so I wouldn’t be able to tell you.

**Comment by tiosgz** [comment]
I've tested it today a bit and this is what it does: https://www.dropbox.com/sh/s8epoxb0sszscda/AABYALuIpOr7eOT2Xsh75N19a?dl=0

It seems to work good only with 0 borders. The best examples of errors with borders are (I think) 2020-04-15-**170106**_1680x1050_scrot.png and 2020-04-15-**165205**_1680x1050_scrot.png

**Comment by dotrosedotnet** [comment]
I wonder if this could be easily implemented through the CSS implementation? Or if it is already? Gosh it would make me happy. I just switched to HLWM this week and I LOVE IT

**Comment by t-wissmann** [comment]
yes, I started the CSS implementation also because I wanted to achieve rounded corners at some point :-) , but it's not implemented so far.

**Comment by dotrosedotnet** [comment]
Yay!! 🥰
